package.json

```diff
@@ -94,7 +94,7 @@
     "run-sequence": "^1.1.4",
     "sinon": "git://github.com/cjohansen/Sinon.JS#b672042043517b9f84e14ed0fb8265126168778a",
     "sinon-chai": "^2.8.0",
-    "ta-scripts": "^2.3.0",
+    "ta-scripts": "^2.3.1",
```

Comment:
This patch version includes a fix to monkey patch relative html file attribute paths.  Previously, it only patched absolute paths.  Stardust uses relative paths for it's assets since it needs to work on `technologyadvice.github.io/stardust`.